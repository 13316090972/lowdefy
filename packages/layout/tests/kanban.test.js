import kanban from '../demo/examples/kanban.yaml';
import runExampleTests from './runExampleTests';

runExampleTests(kanban);
