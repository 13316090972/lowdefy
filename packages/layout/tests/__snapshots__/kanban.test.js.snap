// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic kanban 1`] = `
<div
  className="ant-col {} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
  id="bl-basic kanban23456"
  style={
    Object {
      "alignSelf": undefined,
      "border": "1px dashed #8eccf5",
    }
  }
>
  <div
    id="basic kanban23456"
  >
    <div
      className="ant-row {\\"style\\":[null,{\\"flexWrap\\":\\"nowrap\\",\\"overflow\\":\\"scroll\\",\\"height\\":300}]}"
      id="basic kanban-content23456"
      style={
        Object {
          "border": "1px dashed red",
          "flexDirection": undefined,
          "flexWrap": undefined,
          "overflow": undefined,
        }
      }
    >
      <div
        className="ant-col {}"
        id="bl-list_023456"
        style={
          Object {
            "alignSelf": undefined,
            "border": "1px dashed #8eccf5",
            "flex": "0 0 400px",
          }
        }
      >
        <div
          id="list_023456"
        >
          <div
            className="ant-row {\\"style\\":[null,null]}"
            id="list_0-content23456"
            style={
              Object {
                "border": "1px dashed red",
                "flexDirection": undefined,
                "flexWrap": undefined,
                "overflow": undefined,
              }
            }
          >
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_0_123456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_0_123456"
              >
                item_0_123456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_0_223456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_0_223456"
              >
                item_0_223456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_0_323456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_0_323456"
              >
                item_0_323456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_0_423456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_0_423456"
              >
                item_0_423456
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-col {}"
        id="bl-list_123456"
        style={
          Object {
            "alignSelf": undefined,
            "border": "1px dashed #8eccf5",
            "flex": "0 0 200px",
          }
        }
      >
        <div
          id="list_123456"
        >
          <div
            className="ant-row {\\"style\\":[null,null]}"
            id="list_1-content23456"
            style={
              Object {
                "border": "1px dashed red",
                "flexDirection": undefined,
                "flexWrap": undefined,
                "overflow": undefined,
              }
            }
          >
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_1_123456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_1_123456"
              >
                item_1_123456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_1_423456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_1_423456"
              >
                item_1_423456
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-col {}"
        id="bl-list_223456"
        style={
          Object {
            "alignSelf": undefined,
            "border": "1px dashed #8eccf5",
            "flex": "0 0 400px",
          }
        }
      >
        <div
          id="list_223456"
        >
          <div
            className="ant-row {\\"style\\":[null,null]}"
            id="list_2-content23456"
            style={
              Object {
                "border": "1px dashed red",
                "flexDirection": undefined,
                "flexWrap": undefined,
                "overflow": undefined,
              }
            }
          >
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_2_123456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_2_123456"
              >
                item_2_123456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_2_223456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_2_223456"
              >
                item_2_223456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_2_323456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_2_323456"
              >
                item_2_323456
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-col {}"
        id="bl-list_323456"
        style={
          Object {
            "alignSelf": undefined,
            "border": "1px dashed #8eccf5",
            "flex": "0 0 400px",
          }
        }
      >
        <div
          id="list_323456"
        >
          <div
            className="ant-row {\\"style\\":[null,null]}"
            id="list_3-content23456"
            style={
              Object {
                "border": "1px dashed red",
                "flexDirection": undefined,
                "flexWrap": undefined,
                "overflow": undefined,
              }
            }
          >
            <div
              className="ant-col {\\"style\\":{\\"height\\":150}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_3_123456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_3_123456"
              >
                item_3_123456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_3_223456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_3_223456"
              >
                item_3_223456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":250}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_3_323456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_3_323456"
              >
                item_3_323456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_3_423456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_3_423456"
              >
                item_3_423456
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-col {}"
        id="bl-list_423456"
        style={
          Object {
            "alignSelf": undefined,
            "border": "1px dashed #8eccf5",
            "flex": "0 0 400px",
          }
        }
      >
        <div
          id="list_423456"
        >
          <div
            className="ant-row {\\"style\\":[null,null]}"
            id="list_4-content23456"
            style={
              Object {
                "border": "1px dashed red",
                "flexDirection": undefined,
                "flexWrap": undefined,
                "overflow": undefined,
              }
            }
          >
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_4_123456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_4_123456"
              >
                item_4_123456
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-col {}"
        id="bl-list_523456"
        style={
          Object {
            "alignSelf": undefined,
            "border": "1px dashed #8eccf5",
            "flex": "0 0 400px",
          }
        }
      >
        <div
          id="list_523456"
        >
          <div
            className="ant-row {\\"style\\":[null,null]}"
            id="list_5-content23456"
            style={
              Object {
                "border": "1px dashed red",
                "flexDirection": undefined,
                "flexWrap": undefined,
                "overflow": undefined,
              }
            }
          >
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_5_023456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_5_023456"
              >
                item_5_023456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_5_123456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_5_123456"
              >
                item_5_123456
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-col {}"
        id="bl-list_623456"
        style={
          Object {
            "alignSelf": undefined,
            "border": "1px dashed #8eccf5",
            "flex": "0 0 400px",
          }
        }
      >
        <div
          id="list_623456"
        >
          <div
            className="ant-row {\\"style\\":[null,null]}"
            id="list_6-content23456"
            style={
              Object {
                "border": "1px dashed red",
                "flexDirection": undefined,
                "flexWrap": undefined,
                "overflow": undefined,
              }
            }
          >
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_6_023456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_6_023456"
              >
                item_6_023456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_6_123456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_6_123456"
              >
                item_6_123456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_6_223456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_6_223456"
              >
                item_6_223456
              </div>
            </div>
            <div
              className="ant-col {\\"style\\":{\\"height\\":50}} ant-col-xs-24 ant-col-sm-24 ant-col-md-24"
              id="bl-item_6_323456"
              style={
                Object {
                  "alignSelf": undefined,
                  "border": "1px dashed #8eccf5",
                }
              }
            >
              <div
                className="{\\"style\\":[{\\"background\\":\\"#269\\",\\"border\\":\\"2px solid #00aaee\\",\\"textAlign\\":\\"center\\",\\"fontSize\\":12,\\"color\\":\\"#fff\\"}]}"
                id="item_6_323456"
              >
                item_6_323456
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
